### PR TITLE
MA CK stat fix?

### PR DESCRIPTION
**Pull request by killer2k1**
tribes and time a very weird relationship

### CODE REVIEW

File: base/scripts/code.collector.cs [hunk 1]
```diff
@@ -2,7 +2,14 @@
 $Marker::Flag = "statflag";
 
 function Collector::Clear() {
+    
+    $zadmin::Flag[0] = $Marker::Home;
+    $zadmin::Flag[1] = $Marker::Home;
+    $zadmin::FlagOffStand[0] = getSimTime();
+    $zadmin::FlagOffStand[1] = getSimTime();
+
```
Comment (killer2k1) [inline base/scripts/code.collector.cs:10]: resetting variables at the beginning of every match

File: base/scripts/code.game.cs [hunk 2]
```diff
@@ -905,15 +904,13 @@ function remoteSetArmor(%player, %armorType)
...
       Flag::ResetFlag($teamFlag[%i]);
+      
       $FlagIsDropped[%i] = false;
       $freeze::FlagClient[%i] = 0;
       $freeze::OOB[%i] = false;
       
-      //let clients know flag has returned (flush stack)
-      zadmin::ActiveMessage::All(FlagReturned, %i, 0);
-      //flush stack server
-      Stats::FlagReturned(%i, 0);
```
Comment (killer2k1) [inline base/scripts/code.game.cs:916]: this code was already in objectives.cs - removed

File: base/scripts/code.zbridge.cs [hunk 3]
```diff
@@ -1,47 +1,10 @@
...
-    remoteEval(2048, zAdminActiveMode);
-    zadmin::reset();
-}
-
-function zadmin::reset() {
-    $zadmin::Flag[0] = $Marker::Home;
-    $zadmin::Flag[1] = $Marker::Home;
-    $zadmin::FlagOffStand[0] = getSimTime();
-    $zadmin::FlagOffStand[1] = getSimTime();
-    deleteVariables( "$zadmin::FlagDropTime*" );
```
Comment (killer2k1) [inline base/scripts/code.zbridge.cs:42]: moved to collector clear

File: base/scripts/code.zbridge.cs [hunk 4]
```diff
@@ -1,47 +1,10 @@
...
-//function Stats::ItemReceived( %cl, %item, %count) { Event::Trigger(EventItemReceived, %item, %count); }
-
 $zadmin::WeaponConversion[ "Laser Rifle" ] = "Laser";
 $zadmin::WeaponConversion[ "Chaingun" ] = "Chaingun";
 $zadmin::WeaponConversion[ "Disc Launcher" ] = "Disc";
 $zadmin::WeaponConversion[ "Explosives" ] = "Explosive";
 $zadmin::WeaponConversion[ "Elf Gun" ] = "ELF";
 $zadmin::WeaponConversion[ "Vehicle" ] = "Impact";
 
-function Stats::zAdminActiveMode( %state ) {
```
Comment (killer2k1) [inline base/scripts/code.zbridge.cs:14]: unused function

File: base/scripts/code.zbridge.cs [hunk 5]
```diff
@@ -1,47 +1,10 @@
...
-    if ( %sv != 2048 )
-        return;
-    onClientMessageLegacy.detach();
-    //Event::Detach( eventServerMessage, KillTrak::Parse );
-}
-
-function zadmin::onConnectionAccepted() {
```
Comment (killer2k1) [inline base/scripts/code.zbridge.cs:21]: also believe this was never used

File: base/scripts/objectives.cs [hunk 6]
```diff
@@ -1337,4 +1337,8 @@ function checkObjectives(%this)
   Stats::FlagReturned( %flagTeam, 0 );
   
   GameBase::startFadeIn(%flag);
+  
+  $zadmin::Flag[%flagTeam] = $Marker::Home;
+  $zadmin::FlagOffStand[%flagTeam] = getSimTime();
```
Comment (killer2k1) [inline base/scripts/objectives.cs:1342]: reset these zadmin variables at halftime

File: base/scripts/code.zbridge.cs [hunk 7]
```diff
@@ -1,47 +1,10 @@
-//Event::Attach( eventConnectionAccepted, zadmin::onConnectionAccepted );
```
Comment (killer2k1) [inline base/scripts/code.zbridge.cs:1]: removed commented out lines

**Review by dwayneyuen — APPROVED** [review]

**Pull request merged**